index.html

```diff
@@ -59,119 +70,283 @@ <h2>I'm a Software Developer</h2>
     <section>
         <div class="work-section">
             <div class="recent-work">
-                <h3>My Recent Work</h3>
-                <hr>
+                <h3>My Recent Works</h3>
+                <hr class="recent-hr">
             </div>
             <div class="project-box">
-                <img src="images/Img Placeholder.png" alt="imageholder">
-                <h2>Multi-Post-Stories</h2>
-                <p>
-                    A daily selection of privately personalized
-                    <br> reads; no accounts or sign-ups required.
-                    <br> has been the industry's standard dummy
-                    text ever since the 1500s, when an
-                    <br> unknown printer took a standard dummy
-                    <br> text.
-                </p>
-                <div class="btn">
-                    <ul>
-                        <li>CSS</li>
-                        <li>HTML</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
-                    </ul>
+                <div class="img-holder">
+                    <img id="img1" src="images/Img Placeholder.png" alt="">
+                    <img id="img2" src="images/Img Placeholder (3).png" alt="">
                 </div>
-                <div class="see_more_btn">
-                    <button>See Projects</button>
+                <div class="work-content">
+                    <h2>Multi-Post-Stories</h2>
+                    <p>
+                        A daily selection of privately personalized
+                        <br> reads; no accounts or sign-ups required.
+                        <br> has been the industry's standard dummy
+                        text ever since the 1500s, when an
+                        <br> unknown printer took a standard dummy
+                        <br> text.
+                    </p>
+                    <div class="btn">
+                        <ul>
+                            <li>CSS</li>
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <div class="see_more_btn">
```

Comment:
You did well with your design, kindly implement the following changes in the `recent works` section of your project 😄 

- Please kindly increase font size for the heading `Multi-Post-Stories` to be similar to the design given
- Kindly use the font color given in the Figma template for your project
- Kindly work on the alignment of the text in the `technologies` and  `see project` buttons

Figma design :

![image](https://github.com/Blessman-Newton/Portfolio/assets/20537712/5c0325a6-08bb-4b59-b391-9fa1cab57075)


Your design :

![image](https://github.com/Blessman-Newton/Portfolio/assets/20537712/0e587c21-09e6-48cc-bab2-e7a8cbae2001)

Reply:
Kindly make sure to implement all changes requested by the previous reviewer , the technologies list needs to be aligned to the left , reduce the weight of the paragraph by using initial value for font-weight property and use the same font color #091E42

![Capture d’écran du 2023-07-15 16-51-20](https://github.com/Blessman-Newton/Portfolio/assets/59707859/3d766a48-5550-4cf0-992b-915f05575893)
![Capture d’écran du 2023-07-15 16-51-01](https://github.com/Blessman-Newton/Portfolio/assets/59707859/afee7477-b361-4a32-8ba6-065b8df3e314)